src/index.html

```diff
@@ -22,6 +22,361 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header">
+        <a
+          href="#home"
+          class="logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="Moyo logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav__item">
```